components/record/MediaPresentation.vue

```diff
@@ -4,19 +4,10 @@
   >
     <MediaImage
       v-if="displayImage"
-      :link="imageLink"
       :src="imageSrc"
+      :link="imageLink()"
```

Comment:
If/when imageLink becomes a computed property (see below), the parentheses can be removed.